build_tools/python/cmake_builder/rules.py

```diff
@@ -0,0 +1,149 @@
+## Copyright 2022 The IREE Authors
+#
+# Licensed under the Apache License v2.0 with LLVM Exceptions.
+# See https://llvm.org/LICENSE.txt for license information.
+# SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+"""Helpers that build CMake rules."""
+
+from typing import List, Optional, Sequence
+
+INDENT_SPACES = " " * 2
+
+
+def _get_string_list(values: List[str], quote: bool = True) -> List[str]:
+  if quote:
+    values = [f"\"{value}\"" for value in values]
```

Comment:
I think better to use alternative quotes instead of escaping

```suggestion
    values = [f'"{value}"' for value in values]
```